index.js

```diff
@@ -110,7 +110,7 @@ module.exports = (text, opts) => {
 
 	const contentWidth = widestLine(text) + padding.left + padding.right;
 	const paddingLeft = PAD.repeat(padding.left);
-	const columns = process.stdout.columns;
+	const columns = process.env.STDOUT_COLUMNS || process.stdout.columns;
```

Comment:
Seems like `process.env.COLUMNS` is the convention for this, so use that and add `80` as last fallback:

```js
const columns =  process.stdout.columns || process.env.COLUMNS || 80;
```